openlibrary/macros/StarRatingsStats.html

```diff
@@ -14,7 +14,7 @@
       <span itemprop="ratingValue">$avg</span>
       <span class="dot">·</span>
   </li>
-  <li>
+  <li onclick="location.href='#starRatingSection';">
```

Comment:
```suggestion
  <li>
```
The `onclick` attribute should be added to the previous `<li>` element, [here](https://github.com/internetarchive/openlibrary/blob/48c0eaaef9b1411ff4897514f4c1cb6e8967b702/openlibrary/macros/StarRatingsStats.html#L8).

Reply:
Hey @jimchamp 
So I've changed the `onclick` placement and it works as it should. And I've also changed cursor to "pointer" when you hover over the clickable text (star rating) (picture 1).

Picture 1:
![cursor](https://user-images.githubusercontent.com/110251702/203016227-b92b065e-4fe6-4fcc-a2c4-fda3f6c662da.jpg)

But the issue with scrolling is not happening on my part. I have already applied `scroll-margin-top` in "rating-form.less" (picture 2)

Picture 2:
![scroll](https://user-images.githubusercontent.com/110251702/203016309-1b0a02a4-a910-4f87-a8de-42a112bbfb51.PNG)

So should I do another PR so you can test it again? I've tried changing my browser to mobile version (with lots of different variations of resolutions) and it worked on every one of them.

Also, javascript_test that has failed...is that something that I should fix or be concerned about? (picture 3)

Picture 3:
![test](https://user-images.githubusercontent.com/110251702/203017528-77b88193-2f47-4c34-9901-490152e28213.png)